docs/ci-cd-environment/docker-authentication.md

```diff
@@ -0,0 +1,157 @@
+This means that if you are pulling images from a public Docker Hub repository as an anonymous user, **your Semaphore jobs will be affected by the new DockerHub rate limit**.  
+
+We want to support you through this transition and help you reduce the impact of the Docker Hub rate limit introduction.  
+Feel free to reach out to our support team with any questions that you might have.  
+
+## What should you do to minimize the effect of the rate limit  
+If you have a DockerHub account, **to avoid failing jobs we suggest that you start authenticating your pulls** in your Semaphore configuration.  
+Docker offers a rate limit of 200 pulls per 6 hours for their free plan accounts and unlimited pulls for Pro and Team Docker Hub accounts.  
+Please check instructions on [how to authenticate Docker pulls](#how-to-authenticate-Docker-pulls) in the section below.  
+
+## How can you know if you are hitting the limit
+If you have exceeded the rate limit Docker will throw the `Too Many Requests` error.  
+In the job log check the output of your `docker pull` command, if you have exceeded the rate limit output will be the following:  
+```bash
+Error pulling image <docker-image-name>: Error response from daemon: toomanyrequests: Too Many Requests.`
+```
+
+## How to authenticate Docker pulls
+### Create the Semaphore secret  
+The first step is to store your Docker Hub credentials. You can use [Semaphore secret](https://docs.semaphoreci.com/essentials/using-secrets/) to safely store any credentials and make them available in your projects.  
+
+**Creating a secret from the UI**
+- Click on the organization icon in the top right corner  
+- From the menu select **Settings**  
+- On the left side pick **Secrets**  
+- Click on **New Secret**  
+- Fill in a unique name for your secret  
+- Add the first environment variable: `Variable name: "DOCKER_CREDENTIAL_TYPE", Value: "DockerHub"`  
+- Click on **+ Add another** and add new variable: `Variable name: DOCKER_USERNAME, Value:<your-dockerhub-username>`  
+- Add the third environment variable: `Variable name: DOCKER_PASSWORD, Value:<your-dockerhub-password>`  
+- Click on **Save Secret**  
+
+**Creating a secret through CLI**  
+Before you begin, you'll need to [install the Semaphore CLI][install-cli].  
+
+After connecting to your Semaphore organization update the details in the example command below and run it:  
+```bash
+sem create secret <name-of-your-secret> \
+  -e DOCKER_CREDENTIAL_TYPE=DockerHub \
+  -e DOCKER_USERNAME=<your-dockerhub-username> \
+  -e DOCKER_PASSWORD=<your-dockerhub-password> \
+```
+**Adding a secret to your pipeline YAML**  
+To use your newly created secret in your jobs you have to first attach it.  
+You can attach a secret to individual blocks in your workflow or the whole pipeline.  
+We suggest doing the latter so that it's available to **all jobs** in the workflow.  
+You can achieve this by using `global_job_config` like this:  
```

Comment:
this was used twice, so 'You can achieve this by using `global_job_config` like this:  ' => You can achieve it by using `global_job_config` like this: